src/main/java/org/ffmpeg/android/FfmpegController.java

```diff
@@ -602,33 +604,89 @@ public void squareCrop(File inputFile, File outputFile, int squareSize, ShellCal
         execFFMPEG(commandArray, sc);
     }
 
-    public void trim(File inputFile, File outputFile, ShellCallback sc) throws IOException, InterruptedException
-    {
-        ArrayList<String> c = new ArrayList<String>();
-        c.add(mFfmpegBin);
-        c.add("-i");
-        c.add(inputFile.getAbsolutePath());
-        c.add("-ss");
-        c.add("00:00:00");
-        c.add("-t");
-        c.add("00:00:02");
-
-        //c.add("-f");
-		//c.add("mpeg");
-
-        c.add("-async");
-        c.add("1");
-
-        c.add("-vcodec");
-        c.add("libx264");
-        c.add("-acodec");
-
-        //c.add("-c");
-        c.add("copy");
-
-        c.add(outputFile.getAbsolutePath());
-        execFFMPEG(c, sc);
-    }
+	public void trim(final File inputFile, final File outputFile, final ShellCallback sc) throws IOException, InterruptedException
```

Comment:
This is scary, but it works, and we can bank of it working consistently -- I just still want to test this on a few different devices.